lws10-authn-ssi-cid/index.html

```diff
@@ -0,0 +1,204 @@
+    <script class='remove'>
+      // See https://respec.org/docs/ for how to configure ReSpec
+      var respecConfig = {
+        specStatus: "ED",
+        shortName: "lws-authn-ssi-cid",
+        editors: [{
+          name: "Jesse Wright"
+        }, {
+          name: "Aaron Coburn"
+        }],
+        github: "w3c/lws-protocol",
+        xref: ["web-platform"],
+        group: "lws",
+      };
+    </script>
+  </head>
+  <body>
+    <section id="abstract">
+      <p>
+        This document defines an authentication suite for the Linked Web Storage (LWS) protocol, enabling clients that are able to sign their own identity tokens to integrate with LWS.
+      </p>
+    </section>
+    <section id="sotd">
+      <p>
+        This is an unofficial proposal.
+      </p>
+    </section>
+
+    <section id="introduction">
+      <h2>Introduction</h2>
+      <p>
+        Self-issued identity is important for cases where applications act on their own behalf. This includes autonomous bots as well as server-side scripts, among others.
+        In these cases, the agent is able to securely manage the private portion of a keypair, which it uses to generate signed JSON Web Tokens (JWT).
+        This specification describes how this class of agents can generate end-user credentials that can be used with a Linked Web Storage.
+      </p>
+    </section>
+    <section id="conformance"></section>
+
+    <section id="terminology">
+      <h2>Terminology</h2>
+
+      <p>
+        The terms "authorization server" and "client" are defined by The OAuth 2.0 Authorization Framework [[!RFC6749]].
+      </p>
+
+      <p>
+        The terms "controlled identifier document" and "verification method" are defined by W3C Controlled Identifiers 1.0 [[!CID-1.0]].
+      </p>
+
+      <p>
+        The terms "JSON Web Token (JWT)" and "claim" are defined by JSON Web Token [[!RFC7519]].
+      </p>
+
+      <p>
+        The terms "end-user credential" and "authentication suite" are defined by Linked Web Storage Protocol [[!LWS-PROTOCOL]]
+      </p>
+    </section>
+
+    <section id="serialization">
+      <h2>End-User Credential Serialization</h2>
+
+      <p>
+        A self-issued end-user credential is serialized as a signed JSON Web Token (JWT). In order to use a JWT as an LWS end-user credential, the following additional requirements apply.
+      </p>
+
+      <p>
+        The JWT MUST NOT use "none" as the signing algorithm.
+      </p>
+
+      <p>
+        The JWT MUST use the <code>sub</code> (subject) claim for the LWS subject identifier.
+      </p>
+
+      <p>
+        The JWT MUST use the <code>iss</code> (issuer) claim for the LWS issuer identifier.
+      </p>
+
+      <p>
+        The JWT MUST use the <code>client_id</code> (client ID) claim for the LWS client identifier.
+      </p>
+
+      <p>
+        The claims <code>sub</code>, <code>iss</code>, and <code>client_id</code> MUST all use the same URI value.
```

Comment:
It's not clear to me why this requirement needs to be so strict? What does this constraint accomplish?

[Minimal sufficiency](https://cacm.acm.org/blogcacm/minimal-sufficiency-a-principle-similar-to-end-to-end/) would be ideal here. It's possible this constraint is necessary, but it would be helpful to have some elaboration on how why removing or relaxing this requirement would be insufficient for interop and protocol goals.

(noting also that this requirement is in lws10-authn-ssi-did-key as well)